Studio-Lovelies/GG-JointJustice-Unity
# Feature/Case 1 stories and tests

## Summary

Adds the stories for case 1.
Fixes a few minor bugs so stories can be played.
Adds a test that automatically plays through each story. To ensure the tests do not take forever the test script sets AppearingDialogueController's AppearInstantly property to true. To view the tests as they would be played in game comment out line 55 of the `Case1Tests` script.

## Testing instructions

Play the stories manually (They are in Resources>InkDialogueScripts/Case1)
Run the tests. Check if they pass.

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource: 
  
closes #209
